docs: Deferred accounting settings (#200)
frappe/erpnext_documentation: erpnext_documentation/www/docs/user/manual/en/accounts/deferred-revenue.md

# Deferred Revenue

> **Deferred revenue refers to advance payments a Company receives for products or services that are to be delivered or performed in the future.**

Reviewer: ```suggestion
**Deferred revenue refers to the advance payments a Company receives for products or services that are to be delivered or performed in the future.**
```

It is also known as unearned revenue.

The company that receives the prepayment records the amount as Deferred Revenue on their balance sheet as a liability. Deferred revenue is a liability because it refers to revenue that has not been earned and represents products or services that are owed to a Customer. As the product or service is delivered over time, it is recognized as revenue on the income statement.

## 1. Configuring Deferred Accounting

> Introduced in Version 13

Before you start using deferred accounting you should be aware of the below settings which will give you more control over how you manage your deferred accounting

<img class="screenshot" alt="Deferred Accounting Settings" src="{{docs_base_url}}/assets/img/accounts/deferred-accounting-settings.png">

> 1. **Automatically Process Deferred Accounting Entry:** This setting is enabled by default. In case you don't want the deferred accounting entries to be posted automatically you can disable this setting. If this setting is disabled deferred accounting will have to be processed manually using [Process Deferred Accounting](/docs/user/manual/en/accounts/process-deferred-accounting)

Reviewer: ```suggestion
1. **Automatically Process Deferred Accounting Entry:** This setting is enabled by default. In case you don't want the deferred accounting entries to be posted automatically, you can disable this setting. If this setting is disabled, deferred accounting will have to be processed manually using [Process Deferred Accounting](/docs/user/manual/en/accounts/process-deferred-accounting)
```

> 1. **Book Deferred Entries Based On:** Deferred revenue amount can be booked based on two criteria. The default option here is "Days". If "Days" is selected deferred revenue amount will be booked based on the no of days in each month and if "Months" is selected then it will be booked based on no of months. **For Eg:** If "Days" is selected and $12000 revenue has to be deferred over a period of 12 months then $986.30 will be for the month having 30 days and $1019.17 will be booked for the month having 31 days. If "Months" is selected the $1000 deferred revenue will booked each month irrespective of the number of days in a month

Reviewer: ```suggestion
1. **Book Deferred Entries Based On:** Deferred expense amount can be booked based on two criteria. The default option here is "Days". If "Days" is selected, the deferred expense amount will be booked based on the number of days in each month and if "Months" is selected, then it will be booked based on the number of months. **For Eg:** If "Days" is selected and $12000 expense has to be deferred over a period of 12 months, then $986.30 will be for the month having 30 days and $1019.17 will be booked for the month having 31 days. If "Months" is selected, $1000 deferred expense will be booked each month irrespective of the number of days in a month.
```

> 1. **Book Deferred Entries Via Journal Entry:** By default Ledger Entries are posted directly to book deferred revenue against an invoice. In order to book this deferred amount posting via Journal Entry this option can be enabled

Reviewer: ```suggestion
1. **Book Deferred Entries Via Journal Entry:** By default Ledger Entries are posted directly to book deferred revenue against an invoice. In order to book this deferred amount posting via Journal Entry, this option can be enabled.
```

> 1. **Submit Journal Entries:** This option is applicable only if deferred accounting entries are posted via Journal Entry. By default the Journal Entries for deferred posting are kept in Draft state and a user has to verify those entries and submit them manually. If this option is enabled Journal Entries will be automatically submitted without any user intervention.

Reviewer: ```suggestion
1. **Submit Journal Entries:** This option is applicable only if deferred accounting entries are posted via Journal Entry. By default, the Journal Entries for deferred posting are kept in Draft state and a user has to verify those entries and submit them manually. If this option is enabled, Journal Entries will be automatically submitted without any user intervention.
```
